src/main/scala/higherkindness/rules_scala/common/worker/WorkerMain.scala

```diff
@@ -22,8 +22,6 @@ trait WorkerMain[S] {
     args.toList match {
```

Comment:
This method is getting pretty gnarly. Can we clean this up at all?

Reply:
Not a bad idea, but not something I think we need to do as part of this PR.

Reply:
I'll take note of this as some tech debt cleanup to do in the near future. There's some places where moving code into a well-named function and calling that function will make this far more understanable.